README.md

```diff
@@ -79,7 +80,9 @@ Imagine that you take the code from this question, save it to a file called `foo
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+Nothing would be output to the console.  The reason is that the first "if" statement doesn't execute because howMuchILikeSushi is not >= 3, since it is 2, so nothing gets written to the console.
+
+Then, since "x" has not been defined, it is not greater than 0, which means that the second "if" statement evaluates to false, which means that nothing gets written to the console on the second "if" statement, either.
```

Comment:
Actually, it's worse than that. Not only has `x` not been defined, it hasn't even been declared. As a result, the program will hit an error when it reaches this line.